db/migrate/20190425121602_add_racing.rb

```diff
@@ -15,30 +15,30 @@ def change
       t.datetime :updated_at, limit: 3, null: false
     end
 
-    create_table :bingo_races, id: :uuid do |t|
+    create_table :bingos, id: :uuid do |t|
       t.belongs_to :game, foreign_key: true, null: false
       t.belongs_to :user, foreign_key: true, null: false
 
-      t.string   :status_text, null: false
+      t.integer  :status,      null: false, default: 0
       t.integer  :visibility,  null: false, default: 0
-      t.string   :join_token,  null: false, index: {unique: true}
+      t.string   :join_token,  null: false
       t.string   :notes,       null: true
-      t.string   :card,        null: false
+      t.string   :card_url,    null: true
```

Comment:
Is there a reason to allow `card_url` and `seed` to be null?

Reply:
I thought I remembered us talking about making card and seed optional, but now that I think about it I might be remembering us talking about making them exposed to race entrants from the beginning. I'll revert.